worlds/lingo/test/TestDoors.py

```diff
@@ -0,0 +1,89 @@
+from . import LingoTestBase
+
+
+class TestRequiredRoomLogic(LingoTestBase):
+    options = {
+        "shuffle_doors": "complex"
+    }
+
+    def test_pilgrim_first(self) -> None:
+        assert not self.multiworld.state.can_reach("The Seeker", "Region", self.player)
```

Comment:
```suggestion
        self.assertFalse(self.multiworld.state.can_reach("The Seeker", "Region", self.player))
```
It might be worth adding a `self.can_reach_region` to the `WorldTestBase` but that should be a separate PR.